### PR TITLE
Feater/reward csv download

### PR DESCRIPTION
**Pull request by mdobosz3**
Admin can download the entire order list to a csv file

### CODE REVIEW

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -3,7 +3,23 @@
...
+      #  format.csv { send_data @orders.to_csv, filename: "orders-#{Date.today}.csv" }
+      # end
+
+      respond_to do |format|
+        format.html
+        # render :template => 'admin/orders/index.html.erb'
+        format.csv do
+          response.headers['Content-Type'] = 'text/csv'
+          response.headers['Content-Disposition'] = "attachment; filename=Orders-#{Time.zone.today}.csv"
+          render template: 'admin/orders/export.csv.erb'
```
Comment (piowit) [inline app/controllers/admin/orders_controller.rb:20]: Calling methods with `.` is deprecated according to rspec
```suggestion
          render template: 'admin/orders/export', handlers: [:erb], formats: [:csv]
```

File: app/controllers/admin/orders_controller.rb [hunk 2]
```diff
@@ -3,7 +3,23 @@
...
+      @orders = Order.includes(:employee).all.order(:status)
+
+      # respond_to do |format|
+      #  format.html
+      #  format.csv { send_data @orders.to_csv, filename: "orders-#{Date.today}.csv" }
+      # end
+
+      respond_to do |format|
+        format.html
+        # render :template => 'admin/orders/index.html.erb'
```
Comment (piowit) [inline app/controllers/admin/orders_controller.rb:16]: i think this is not needed here, as we are only rendering csv

**Review by piowit — COMMENTED** [review]
i've added my suggestions

File: app/controllers/admin/orders_controller.rb (hunk 1 above)
Comment (mdobosz3) [reply app/controllers/admin/orders_controller.rb]: Indeed rubocop reports this entry as incorrect

File: app/controllers/admin/orders_controller.rb (hunk 2 above)
Comment (mdobosz3) [reply app/controllers/admin/orders_controller.rb]: Of course it was a draft, now there are no unnecessary entries anymore :)

**Pull request merged**